app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,7 @@
   <main>
```

Comment:
It is good to update the `title` element, above on line 4, so that your web app has a customized title